bokehjs/test/integration/widgets.ts

```diff
@@ -217,6 +217,16 @@ describe("Widgets", () => {
     await display(obj, [320, 120])
   })
 
+  it("should allow TeX on Divs with delimiters between new lines", async () => {
+    const obj = new Div({
```

Comment:
It's a waste of visual integration tests for this. This should have been solely tested on unit-level.